app/controllers/review_queues_controller.rb

```diff
@@ -36,6 +48,24 @@ def submit
     render json: { status: 'ok' }
   end
 
+  def item
+    @item = ReviewItem.find(params[:item_id])
+    render 'queue.html.erb'
+  end
+
+  def reviews
+    @all = params[:all] == '1'
+
+    @reviews =
+      if @all
+        ReviewResult
+      else
+        ReviewResult.where(user: current_user)
+      end
+      .joins(:item).where(review_items: { review_queue_id: @queue.id }, review_results: { user: current_user })
```

Comment:
Duplicate logic - you're checking an `if` above to see whether it's user-scoped or all, but this line applies a `where(review_results: { user: current_user })` - you'll always get your own reviews here.

Reply:
Oops, copy/paste mistake I think